### PR TITLE
Link lemp9 to tech-docs.system76.com

### PR DESCRIPTION
**Pull request by jackpot51**

### CODE REVIEW

**Comment by jackpot51** [comment]
That's fine, do you want to make edits to this PR?

**Comment by ahoneybun** [comment]
I can make edits.

**Comment by WatchMkr** [comment]
@ahoneybun is the purpose of offline for the customer to print a reference while their computer is taken apart?

**Comment by ahoneybun** [comment]
Yes that would be the main reason though not a lot of people know about the printing of web pages.

**Comment by WatchMkr** [comment]
Because this document will contain all models, and links to exterior sources in a more modern way, it would be better to describe in the documentation that it's printable. mdBook also formats nicely for printing.

PDFs won't be sustainable long-term.

BTW, nice work on these docs everyone. They're getting really good.

File: _articles/service-manuals.md [hunk 1]
```diff
@@ -15,17 +15,17 @@ section: repairs-returns
 
 ---
 
-| Laptops      | Service Manual |
-| -------------| ---------------|
-| Adder WS     | [addw1](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Adder/addw1-service-manual.pdf)
+| Laptops      | PDF | Tech Docs |
+| -------------| ----| ----------|
+| Adder WS     | [addw1](https://github.com/system76/docs/blob/gh-pages/service-manuals/pdfs/Adder/addw1-service-manual.pdf) | [addw1](https://tech-docs.system76.com/models/addw1/repairs.html)
```
Comment (jackpot51) [inline _articles/service-manuals.md:20]: Should be https://tech-docs.system76.com/models/addw1/README.html

**Comment by jackpot51** [comment]
This is really not what was intended. The case of printing the pages is handled by printing from the website. The offline copy is incomplete in features and large in size.

We should link to the tech-docs here, alone. If a user still needs an offline copy, and is unable to get it from printing the tech-docs to a file, then I believe there is still work we need to do in tech-docs to make the PDF output better

**Comment by ahoneybun** [comment]
Going to use the merge Pop Docs PR to make this change.

**Pull request closed without merging**